dev-docs/bidders/adsyield.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: bidder
+title: AdsYield
+description: Prebid AdsYield Bidder Adaptor
+pbjs: true
+biddercode: adsyield
+aliasCode: admixer
+media_types: video
+gdpr_supported: true
+usp_supported: true
+---
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name          | Scope    | Description                                                                                                      | Example                                | Type     |
+|---------------|----------|------------------------------------------------------------------------------------------------------------------|----------------------------------------|----------|
+| `zone`        | required | The unique identifier of the ad placement. Could be obtained from the AdsYield UI or from your account manager. | "e5ff8e48-4bd0-4a2c-9236-55530ab8981d" | `string` |
+| `kvTargeting` | optional | Key/Value - a pair of the unique values that will be used for the custom targeting option.                       | {key1: value2, key2: value2}           | `object` |
```

Comment:
Let's talk about this kvTargeting a little. Prebid is trying to get pubs to support First Party Data as a standard instead of passing data separately into many bidders. Your bid adapter(s) should be reading the `ortb2` object as a source of FPD. See https://docs.prebid.org/features/firstPartyData.html